src/AbstractSQLRules2SQL.ts

```diff
@@ -142,7 +142,9 @@ export const isNumericValue = (
 		type === 'Ceiling' ||
 		type === 'Count' ||
 		type === 'Average' ||
-		type === 'Sum'
+		type === 'Sum' ||
+		isDateValue(type) ||
+		isDurationValue(type)
```

Comment:
These are not numeric values and it's actually more accurate to say that you can do `number + number` or `date + number` but not `date + date` but you can do `date - date`, see https://www.postgresql.org/docs/14/functions-datetime.html for a list of the operators for date/time types in postgres